static/templates/actors/character/tabs/crafting.hbs

```diff
@@ -44,7 +44,7 @@
                     </li>
                     <!-- Add formula items for each formula level -->
                     {{#each section as |craftedItem i|}}
-                        <li class="item formula-item" data-formula-lvl="{{lvl}}" data-item-id="{{craftedItem.uuid}}" data-is-formula="true">
+                        <li class="item formula-item" data-formula-lvl="{{lvl}}" data-item-id="{{craftedItem.uuid}}" data-item-name="{{craftedItem.name}}" data-is-formula="true">
```

Comment:
Can this depend entirely on the UUID?

Reply:
@stwlam I think there is a bigger pull request open for work on the crafting tab, which is #6324 and there as far as I can tell it also adds the confirmation popup, so I would just close this smaller pull request and let the larger work fix this up. Also adding a comment in the issue mentioned above, to inform that this is worked on.